src/monster.cpp

```diff
@@ -618,24 +617,25 @@ int monster::print_info( const catacurses::window &w, int vStart, int vLines, in
     // display sense
     std::string senses_str = "--";
     if( sees( g->u ) ) {
-        senses_str = _( "It is aware of your presence" );
+        senses_str = _( "Aware of your presence" );
     } else {
-        senses_str = _( "It hasn't noticed you" );
+        senses_str = _( "Unaware of your presence" );
```

Comment:
The difference between the two lines (seen / not seen) is now only two letters, which is easy to miss. The current lines are much easier to distinguish.

Are there any reasons for changing this?